server/TODO.md

```diff
@@ -7,6 +7,7 @@
 - [ ] API compression with https://github.com/expressjs/compression
 - [ ] Add a prop to Flight that indicates if the flight editable (14 days limit)
 - [ ] Use node cluster for scoring to prevent slow API response during results calculation
+- [ ] Find a way to add links to news items without injecting html (List of links with title?)
```

Comment:
HTML, bspw. Sonderzeichen als solches werden eh alle escapen und so in der DB gespeichert. Das gleiche passiert auch bei den Kommentaren. Bei den Kommentaren wandle ich dann ectl. Sonderzeichen beim lesen aus der DB wieder zurück.

Reply:
Vue rendert kein html aus variablen. Das kann man aktivieren, aber es wird davon abgeraten.
[https://v3.vuejs.org/guide/security.html#html-content](https://v3.vuejs.org/guide/security.html#html-content)